File: windows/RNTrackPlayer/Logic/Events.cs
namespace TrackPlayer.Logic
{

    public static class Events
    {
        public const string ButtonPlay = "remote-play";
        public const string ButtonPause = "remote-pause";
        public const string ButtonStop = "remote-stop";
        public const string ButtonSkipNext = "remote-next";
        public const string ButtonSkipPrevious = "remote-previous";
        public const string ButtonSeekTo = "remote-seek";
        public const string ButtonJumpForward = "remote-jump-forward";
        public const string ButtonJumpBackward = "remote-jump-backward";

        public const string PlaybackState = "playback-state";
        public const string PlaybackTrackChanged = "playback-track-changed";
        public const string PlaybackQueueEnded = "playback-queue-ended";
        public const string PlaybackError = "playback-error";
    }

}

File: windows/RNTrackPlayer/Logic/Events.cpp
#include "pch.h"
#include "Logic/Events.h"

using namespace winrt::RNTrackPlayer;

const char* Events::ButtonPlay = "remote-play";
const char* Events::ButtonPause = "remote-pause";
const char* Events::ButtonStop = "remote-stop";
const char* Events::ButtonSkipNext = "remote-next";
const char* Events::ButtonSkipPrevious = "remote-previous";
const char* Events::ButtonSeekTo = "remote-seek";
const char* Events::ButtonJumpForward = "remote-jump-forward";
const char* Events::ButtonJumpBackward = "remote-jump-backward";

const char* Events::PlaybackState = "playback-state";
const char* Events::PlaybackTrackChanged = "playback-track-changed";
const char* Events::PlaybackQueueEnded = "playback-queue-ended";
const char* Events::PlaybackError = "playback-error";
